storage-access.bs

```diff
@@ -302,6 +271,30 @@ To the [=parse a sandboxing directive=] algorithm, add the following under step
 <li>The [=sandbox storage access by user activation flag=], unless <var ignore>tokens</var> contains the <dfn export attr-value for=iframe/sandbox>allow-storage-access-by-user-activation</dfn> keyword.
 </ul>
 
+<h2 id="permissions-integration">Permissions Integration</h2>
+
+The Storage Access API defines a [=powerful feature=] identified by the [=powerful feature/name=] "<dfn export permission><code>storage-access</code></dfn>".
+
+<h3 id="permissions-key-generate">Changes to generate a permission store key</h3>
+
+Update the generate a permission store key algorithm to take the [=powerful feature/name=] |name|. Prepend the following steps to it:
```

Comment:
Is this meant to be the final text? As I don't think we want to patch algorithms in this way.

Reply:
Well this would go into the permissions spec eventually, any suggestions on how to do it differently?

Reply:
That sounds fine, but then let's have a PR against Permissions and land them around the same time.

Reply:
This no longer applies 🎉